Introduce a new plugin of tool: MQTT-Tools.

wanghaEMQ opened the pull request:
# Plugin Submission Form

## 1. Metadata
<!--
Please provide the following metadata of your plugin to make it easier for the reviewer to check the changes.
  - Plugin Author : The author of the plugin which is defined in your manifest.yaml
  - Plugin Name   : The name of the plugin which is defined in your manifest.yaml
  - Repository URL: The URL of the repository where the source code of your plugin is hosted
-->

- **Plugin Author**: wangha
- **Plugin Name**: mqtt
- **Repository URL**: https://github.com/wanghaEMQ/dify-plugin-mqtt

## 2. Submission Type

- [X] New plugin submission
- [ ] Version update for existing plugin

## 3. Description
<!-- Please briefly describe the purpose of the new plugin or the updates made to the existing plugin -->
To communicate with a MQTT Broker.

## 4. Checklist

- [X] I have read and followed the Publish to Dify Marketplace guidelines
- [X] I have read and comply with the Plugin Developer Agreement
- [X] I confirm my plugin works properly on both Dify Community Edition and Cloud Version
- [X] I confirm my plugin has been thoroughly tested for completeness and functionality
- [X] My plugin brings new value to Dify

## 5. Documentation Checklist

Please confirm that your plugin README includes all necessary information:

- [X] Step-by-step setup instructions
- [X] Detailed usage instructions
- [X] All required APIs and credentials are clearly listed
- [X] Connection requirements and configuration details
- [x] Link to the repository for the plugin source code

## 6. Privacy Protection Information

Based on Dify Plugin Privacy Protection [Guidelines](https://docs.dify.ai/plugins/publish-plugins/publish-to-dify-marketplace/plugin-privacy-protection-guidelines):

No privacy information required.

### Data Collection
<!-- Does your plugin collect any user personal data? If yes, please list what types of user personal data are being collected according to the Plugin Privacy Protection Guidelines (for example: Email address, IP address, Age, etc) -->

No data collected

### Privacy Policy

- [X] I confirm that I have prepared and included a privacy policy in my plugin package based on the Plugin Privacy Protection Guidelines

LogicOber commented:
@wanghaEMQ Hi 👋 Thank you for your contribution! It looks like a very interesting plugin. Do you have any chatflows/workflows, showcases, or blogs that use this plugin to share?

LogicOber reviewed (changes requested):
Thank you so much for your pull request on GitHub! 😊 Your contributions are greatly appreciated!

Please ensure that:
- [ ] The PRIVACY.md file includes comprehensive information about data handling
- [ ] All content in PRIVACY.md, README.md, and code 👉 remains in English only 👈 (e.g., when you use `yield self.create_text_message`)
- [ ] Please ensure that the `label` for different languages in ALL THE YAML FILES are written in their corresponding languages.

Looking forward to your updates! 💯

crazywoola commented:
Please fix the errors in CI and submit it again :)

wanghaEMQ commented:
@LogicOber Hi LogicOber. Thanks for your fast and warm reply. I have made some updates to this plugin, https://github.com/langgenius/dify-plugins/pull/617. I think it should work now. 😄 

what changed:

* Updated the PRIVACY.md
* Updated the labels for pt_br.
* Fixed the requirements.txt
* Updated the plugin name.

Furthermore, I found a cool platform difyshare to share dify apps. So... I create an app and upload it to difyshare to show how this plugin works. https://difyshare.com/flow/68403da90002d5f6488a
